jetstream/cli.py

```diff
@@ -792,8 +812,11 @@ def rerun(
 @experiment_slug_option
 def export_statistics_to_json(project_id, dataset_id, bucket, experiment_slug):
     """Export all tables as JSON to a GCS bucket."""
-    for slug in experiment_slug:
-        export_statistics_tables(project_id, dataset_id, bucket, slug)
+    if bucket is None:
+        click.echo("No bucket specified. Analysis results won't be exported to GCS.")
```

Comment:
Is it better to use `click.echo` over `logger.warning`/`info`? I don't think we've used `click.echo` elsewhere.

Reply:
yeah, right

Reply:
I don't think we'd want a `ValueError` in this case though since any run with bucket set to `None` on Argo will call these entrypoints (just based on how the workflow is defined). It would be potentially confusing for users to see their runs fail in those cases.